specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/examples/RedisEnterpriseAccessPolicyAssignmentCreate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "api-version": "2024-09-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f",
+    "resourceGroupName": "rg1",
+    "clusterName": "cache1",
+    "databaseName": "default",
+    "accessPolicyAssignmentName": "defaultTestEntraApp1",
+    "parameters": {
+      "properties": {
+        "accessPolicyName": "default",
+        "user": {
+          "objectId": "6497c918-11ad-41e7-1b0f-7c518a87d0b0"
```

Comment:
@TimLovellSmith, @joroda In OSS we pass the ObjectId as straight property under Properties, here we decided to go as a property of user. The idea is that in the future we will support different types of users. Does this sound good or should we just pass objectId as property?

Reply:
I am totally okay with having user here, I think it does little harm.